tracer/test/snapshots/Iast.DatabaseSourceInjection.AspNetCore5.DbOnly.verified.txt

```diff
@@ -0,0 +1,50 @@
+﻿[
```

Comment:
No vuln detected because we don't want stored XXS to tricker for ssrf? Could be nice to update the PR text to explain that (or some other place?), it had me puzzled for a while.

Reply:
No vuln detected because tainted source is purely database and it's not a XSS nor SQLI, so no injection should be raised.